### PR TITLE
Task 2-Snazzy Button

### PR DESCRIPTION
**Pull request by kwarnerredhat**

### CODE REVIEW

File: src/api/SnazzyButton.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+
+interface SnazzyButtonProps extends ButtonProps {
+  isSnazzy?: boolean;
+}
+const useStyles = createUseStyles({
+  snazzyButton: {
```
Comment (Andrewgdewar) [inline src/api/SnazzyButton.tsx]: Read what the task is asking you to do. 
You have not used the given styles.

File: src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ export default () => {
             onChange={(value) => setNewUser({ ...newUser, isCool: value })}
             isChecked={newUser.isCool}
           />
-          <Button type='submit'>Submit</Button>
+          <Button type='submit'>Snazzy</Button>
```
Comment (Andrewgdewar) [inline src/pages/Landing/index.tsx]: This is not using your new component. You just changed the "Submit" buttons text to "Snazzy"

File: src/api/SnazzyButton.tsx (hunk 1 above)
Comment (kwarnerredhat) [reply src/api/SnazzyButton.tsx]: what styles was given it just said create a component are you talking about css

Comment (Andrewgdewar) [reply src/api/SnazzyButton.tsx]: It has a link to the Styles (css) that you need to use.

**Comment by Andrewgdewar** [comment]
Closing as the approach was not correct, and this PR includes changes from the colorFix PR.

**Pull request closed without merging**